New version adapted to nullable columns and null handling in multi-stage engine (#304)
pinot-contrib/pinot-docs: configuration-reference/table.md

| columnMinMaxValueGeneratorMode             | <p>Generate min max values for columns. Supported values:<br><code>NONE</code> - do not generate for any columns<br><code>ALL</code> - generate for all columns<br><code>TIME</code> - generate for only time column<br><code>NON_METRIC</code> - generate for time and dimension columns</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |

> | nullHandlingEnabled                        | (deprecated, use `enableColumnBasedNullHandling` in [schema](schema.md) instead) Read more about this in the [null handling section](../developers/advanced/null-value-support.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

Reviewer: ```suggestion
| nullHandlingEnabled                        | (deprecated, use `enableColumnBasedNullHandling` in [schema](schema.md)) For more information, see the [null handling section](../developers/advanced/null-value-support.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
